tr/language.json

```diff
@@ -3,960 +3,1119 @@
 		"DE": "Raid-Boss gespeichert!",
 		"EN": "Raid boss saved!",
+		"IT": "Boss del Raid salvato!",
 		"PT-BR": "Chefe da raid salvo!"
 	},
 	"not_supported" : {
 		"NL": "Niet ondersteund",
 		"DE": "Nicht unterstützt",
 		"EN": "Not supported",
+		"IT": "Non compatibile",
 		"PT-BR": "Não compatível"
 	},
 	"share" : {
 		"NL": "Delen",
 		"DE": "Teilen",
 		"EN": "Share",
+		"IT": "Condividi",
 		"PT-BR": "Compartilhar"
 	},
 	"share_with" : {
 		"NL": "Delen met",
 		"DE": "Teilen mit",
 		"EN": "Share with",
+		"IT": "Condividi con",
 		"PT-BR": "Compartilhar com"
 	},
 	"set_gym_name_and_team" : {
 		"NL": "Optioneel - Gym naam and Gym Team:",
 		"DE": "Optional - Arena Name und Arena Team:",
 		"EN": "Optional - Gym name and Gym Team:",
+		"IT": "Opzionale - Nome della Palestra e Team della Palestra",
 		"PT-BR": "Opcional - Nome do gínasio e nome do time"
 	},
 	"set_gym_name_command" : {
 		"NL": "/gym <code>Naam van de Gym</code>",
 		"DE": "/gym <code>Name der Arena</code>",
 		"EN": "/gym <code>Name of the Gym</code>",
+		"IT": "/gym <code>Nome della Palestra</code>",
 		"PT-BR": "/gym <code>Nome do ginásio</code>"
 	},
 	"set_gym_team" : {
 		"NL": "Optioneel - zet Gym en Team",
 		"DE": "Optional - Arena Team setzen",
 		"EN": "Optional - set Gym and Team",
+		"IT": "Opzionale - imposta Palestra e Team",
 		"PT-BR": "Opcional - escolha o gínasio e time"
 	},
 	"set_gym_team_command" : {
 		"NL": "/team <code>Mystic/Valor/Instinct/Blauw/Rood/Geel</code>",
 		"DE": "/team <code>Mystic/Valor/Instinct/Blau/Rot/Gelb</code>",
 		"EN": "/team <code>Mystic/Valor/Instinct/Blue/Red/Yellow</code>",
+		"IT": "/team <code>Saggezza/Valore/Istinto/Blu/Rosso/Giallo</code>",
 		"PT-BR": "/team <code>Mystic/Valor/Instinct/Azul/Vermelho/Amarelo</code>"
 	},
 	"start_date_time" : {
 		"NL": "Starttijd gezet op",
 		"DE": "Startzeit gesetzt auf",
 		"EN": "Starttime set to",
+		"IT": "Orario di inizio impostato alle",
 		"PT-BR": "Hora de começar determinada para"
 	},
 	"end_time" : {
 		"NL": "Eindtijd gezet op ",
 		"DE": "Ablaufzeit gesetzt auf ",
 		"EN": "Endtime set to ",
+		"IT": "Orario di fine impostato alle",
 		"PT-BR": "Hora de término determinada para "
 	},
 	"raid_select_date" : {
 		"NL": "Selecteer de <b>datum</b> van de raid:",
 		"DE": "<b>Datum</b> des Raids auswählen:",
 		"EN": "Select the <b>date</b> of the raid:",
+		"IT": "Seleziona la <b>data</b> del Raid:",
 		"PT-BR": "Select the <b>date</b> of the raid:"
 	},
 	"raid_select_hour" : {
 		"NL": "Selecteer het <b>uur</b> van de raid:",
 		"DE": "<b>Stunde</b> des Raids auswählen:",
 		"EN": "Select the <b>hour</b> of the raid:",
+		"IT": "Seleziona l'<b>ora</b> del Raid:",
 		"PT-BR": "Select the <b>hour</b> of the raid:"
 	},
 	"raid_select_start_time" : {
 		"NL": "Selecteer de <b>start tijd</b> van de raid:",
 		"DE": "<b>Startzeit</b> des Raids auswählen:",
-		"EN": "Select the <b>start time</b> of the raid:",
+		"IT": "Seleziona l'<b>orario di inizio</b> del Raid:",
```

Comment:
Please re-add the English translation ;)

Reply:
Sure, can you please give me an example of the unnecessary whitespaces/tabs? I seem to can't see them!

Ps: sorry, I didn't notice that I have deleted that translation, must've been a copy/paste mistake!